docs/src/get-started.md

```diff
@@ -431,4 +431,4 @@ In order to archive an album, the [`archive` command](user-guides/cli.md#archive
 
 Please check the [contributing guidelines](https://github.com/steilerDev/icloud-photos-sync/blob/main/CONTRIBUTING.md) to learn how to engage with this project. The document outlines the bug reporting, feature and support request process for this tool.
 
-Consider supporting the development efforts by [sponsoring the author](https://github.com/sponsors/steilerDev).
\ No newline at end of file
+Consider supporting the development efforts by [sponsoring the author](https://github.com/sponsors/steilerDev).
```

Comment:
my Markdown prettier on file save removed the trailing space on this line. hope that's fine.